### ISSUE TITLE
Shows a different statement for each if statement... checkbox 2 and 3

### ISSUE THREAD

**Issue by InHoh** [KALoneNinja/Evaluation-Extension#15]
The task's full text is 

> Shows a different message for each if statement and possible value of the answer variable (where each message must differ more than just displaying the different value of answer).

From a teacher point of view the current suggestions leave too little for the student to figure out on their own! I'd like the suggestion to point back to the article just before this project which contains the necessary information, and then just give a hint. As long as the evaluator leaves their link in tips and thanks it should be easy for the student to ask in case the article is not enough for them. 

And from the evaluator point of view, if they don't have to input the arguments in floor random and tell which if-statements are not part of the range, they save time! 

Of course, a less knowledgeable evaluator would get help on this evaluation if functionality was included to show possible values of 'answer' with different input. But then the project might also contain round instead of floor or dispense with either, so the feedback might not be meaningful.

The text, if one inserts the faulty value(s)

> There is an if statement with a condition that is never true. floor(random(**1, 5**)) The value the statement(s) use: **0, 5, and 6** 

The generated text will, for the random(1, 5) with conditionals starting at 1 and ending at 5 be:

> The problem here is that random(1, 5) at its max returns ~4.9999 and at its minimum will return 1.0000. 1.0000 rounded down with floor is 1 and 4.9999 rounded down is 4. Your if statement for the value 5 doesn't fit within that range. Perhaps you could extend the range floor(random()) will return?

Suggested change (you may want to include Random numbers as a link https://www.khanacademy.org/computing/computer-programming/programming/logic-if-statements/a/checking-random-numbers but the student ought to be able to find it on their own.):

You must change the input to the random function so that all the texts have a fair chance of being displayed. Your conditionals do not quite fit the range of possible 'answer's. Can you figure out which of your expected 'answer'(s) are not in that range? The article Random numbers just before this project explains both which numbers random(1, 5) can return, and also what floor will do to a number. 

However, the last sentence must be added only once if the TWO last buttons are checked.
If there correct range has holes, and the actual range is exceeded, which I have seen twice in the last few days, you'll currently get something like:

> The problem here is that `random(1, 5)` at its max returns ~4.9999 and at its minimum will return 1.0000. 1.0000 rounded down with floor is 1 and 4.9999 rounded down is 4. Your if statement for the value 0, 5, and 6 doesn't fit within that range. Perhaps you could extend the range `floor(random())` will return?
> 
> The problem here is that `random(1, 5)` at its max returns ~4.9999 and at its minimum will return 1.0000. 1.0000 rounded down with floor is 1 and 4.9999 rounded down is 4. Your if statements do not completely cover that range, specifically for 1 and 4. Try adding more if statements.

The second text should be something like

"All possible values of 'answer' need to have its own text displaying. Which value(s) of 'answer' is/are missing in your project?"

(Then the sentence "The article Random numbers just before this project explains both which numbers random(1, 5) can return, and also what floor will do to a number. " should be added at the end.)

I have seen projects where the student thinks the range of floor(random(1, 5)) is 5 inclusive, 1 exclusive, but the most common reason for missing values is either 
that the student does not follow the natural order of numbers, and misses one by accident,
or 
that they misunderstand the requirement for at least two else to mean they can get away with that no matter what the range is.

**Comment by KALoneNinja** [comment]
Should be fixed soon
